src/widgets/configure/configure.ts

```diff
@@ -10,14 +9,14 @@ import { noop } from '../../lib/utils';
  * A list of [search parameters](https://www.algolia.com/doc/api-reference/search-api-parameters/)
  * to enable when the widget mounts.
  */
-export type ConfigureWidgetParams = PlainSearchParameters;
+export type ConfigureWidgetParams = ConfigureConnectorParams['searchParameters'];
```

Comment:
Isn't it the same? As`ConfigureConnectorParams['searchParameters'] === PlainSearchParameters`, the naming makes more sense anyway tho

Reply:
Yes, it's the same, but if we add any comment to search parameters option, it will be visible here automatically